modules/space/translation/spicetranslation.cpp

```diff
@@ -121,6 +132,10 @@ SpiceTranslation::SpiceTranslation(const ghoul::Dictionary& dictionary)
         }
     }
 
+    if (p.fixedDate.has_value()) {
```

Comment:
No need to do this here (see comment above).  The _fixedDate assignment is at the bottom of the function which will trigger the `onChange` as well